management/sql/client.go

```diff
@@ -25,22 +25,22 @@ const (
 	DatabaseStateCreating = "Creating"
 )
 
-// SqlDatabaseClient defines various database CRUD operations.
+// Client defines various database CRUD operations.
```

Comment:
Will do.
However; I must warn you that [on my setup, at least], go lint cries that using it as sql.SQLDatabaseClient in other packages is needless repetition; that's why I made it into just Client in the first place...